src/gui/SetupTotpDialog.h

```diff
@@ -35,25 +35,20 @@ class SetupTotpDialog : public QDialog
     Q_OBJECT
 
 public:
-    explicit SetupTotpDialog(DatabaseWidget* parent = nullptr, Entry* entry = nullptr);
-    ~SetupTotpDialog();
-    void setSeed(QString value);
-    void setStep(quint8 step);
-    void setDigits(quint8 digits);
-    void setSettings(quint8 digits);
-
-private Q_SLOTS:
-    void toggleDefault(bool status);
-    void toggleSteam(bool status);
+    explicit SetupTotpDialog(QWidget *parent = nullptr, Entry* entry = nullptr);
```

Comment:
`QWidget*`

This class should also be called `TotpSetupDialog`.

Reply:
I agree with the name, I did consider that change